posydon/utils/posydonerror.py

```diff
@@ -1,5 +1,13 @@
 """The POSYDON exception class and subclasses for more specific errors."""
 
+
+__authors__ = [
+    "Camille Liotine <cliotine@u.northwestern.edu>",
```

Comment:
Add Eirini as a primary author, as she did most of this code development with me.